testsuite/mpitests/test_multiple_synapses_spatial_networks.sli

```diff
@@ -38,21 +38,38 @@
 (unittest) run
 /unittest using
 
-
 /collocated_synapse_params
   [<< /weight -3. /delay 1.0 /synapse_model /static_synapse >>
    << /weight  3. /delay 1.3 /synapse_model /static_synapse >>
-   << /weight  2. /delay 1.9 /synapse_model /stdp_synapse >> ] def
+   << /weight  2. /delay 1.9 /synapse_model /stdp_synapse >> ]
+def
 
-[-0.25 0.25 0.25] Range
-  { /x Set [0.25 -0.25 -0.25] Range { x exch 2 arraystore } Map } Map 1 Flatten
-  /pos Set
+/pos
+  [-0.25 0.25 0.25] Range
+  {
+    /x Set
+    [0.25 -0.25 -0.25] Range
+    {
+      x exch 2 arraystore
+    }
+    Map
+  }
+  Map
+  1 Flatten
+def
 
-% There are 9 positions in positions vector, so 9 neurons per source and target
-/num_src 9 def
-/num_trgt 9 def
+% Set number of sources and targets according to number of positions
+/num_src pos length def
+/num_trgt pos length def
 /num_conns 4 def % we use one_to_one
```

Comment:
Could you also remove the `% we use one_to_one` comment, it is a copy-paste leftover from `test_multiple_synapses.sli`. And maybe update the comment to state that it is number of connections per target.

Reply:
Done.